### PR TITLE
Fixes missing lookup var in ticket lock renews

### PR DESCRIPTION
**Pull request by clonemeagain**
Rebased (well, rewritten) https://github.com/clonemeagain/osTicket-1.8/commit/9c2d8a0ebdc4ade3003a7df69c9f6cb80839d011 for 1.7

As requested: https://github.com/osTicket/osTicket-1.8/pull/341#issuecomment-31010725

Referencing pull request: https://github.com/osTicket/osTicket-1.8/pull/341

### CODE REVIEW

File: include/ajax.tickets.php [hunk 1]
```diff
@@ -237,7 +237,7 @@ function renewLock($tid, $id) {
     function releaseLock($tid, $id=0) {
         global $thisstaff;
 
-        if($id && is_numeric($id)){ //Lock Id provided!
+        if($tid && is_numeric($tid)){ //Lock Id provided!
```
Comment (greezybacon) [inline include/ajax.tickets.php:240]: This should remain as it was. If no lock id (`$id`) was provided, the `else` part of the `if` block should be executed (below)

Comment (clonemeagain) [inline include/ajax.tickets.php:240]: Good spotting chief.  :-)

Comment (greezybacon) [inline include/ajax.tickets.php:240]: Would you mind rebasing it, or would you like me to?

**Comment by greezybacon** [comment]
I opened #878 with all the code changes except the one I commented on

**Pull request closed without merging**